src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -416,7 +474,21 @@ const Notebooks = _.flow(
             onDismiss: () => this.setState({ deletingNotebookName: undefined }),
             onSuccess: () => {
               this.setState({ deletingNotebookName: undefined })
-              this.refresh()
+              this.refreshNotebooks()
+            }
+          }),
+          //todo if status is deleting consider not opening the panel
+          !!openGalaxyConfigDrawer && h(NewGalaxyModal, {
```

Comment:
In order to make the fade animation work for drawers, we need to always render them, and control their visibility with `isOpen`